packages/ui/__stories__/Slider.stories.tsx

```diff
@@ -0,0 +1,235 @@
+    </>
+  )
+}
+
+export const SliderWithError = () => {
+  const [value, onChange] = useState<number>(0)
+  return (
+    <div>
+      <Slider name="name" label="Slider's label" value={value} onChange={onChange} min={0} max={100} error="This value is invalid" />
+    </div>
+  )
+}
+
+export const SliderWithHelperText = () => {
+  const [value, onChange] = useState<number>(0)
+  return (
+    <div>
+      <Slider name="name" label="Slider's label" value={value} onChange={onChange} min={0} max={100} helperText="Helper text" />
+    </div>
+  )
+}
+
+export const SliderWithMarks = () => {
+  const [value, onChange] = useState<number>(0)
+  return (
+    <div>
+      <Slider
+        name="name"
+        label="Slider's label"
+        value={value}
+        onChange={onChange}
+        min={0}
+        max={100}
+        helperText="Helper text"
+        marks={[
+          { value: 0, label: '0GB' },
+          { value: 30, label: '1GB' },
+          { value: 70, label: '4GB' },
+          { value: 100, label: '100GB' },
+        ]}
+      />
+    </div>
+  )
+}
+
+export const SliderInFormik = () => {
+  type Values = {
+    ram: number
+  }
+  return (
+    <Formik<Values>
+      initialValues={{
+        ram: 3,
+      }}
+      initialErrors={{
+        ram: 'Server Error: Invalid ram value for this cloud provider',
+      }}
+      validate={(values) => {
+        const errors: Partial<{ [key in keyof Values]: string }> = {
+          ram: values.ram === 4 ? 'RAM is invalid' : undefined,
+        }
+
+        return errors
+      }}
+      onSubmit={(values) => logger.log('submit', values)}
+    >
+      {({ values }) => (
+        <Form>
+          Values: {JSON.stringify(values)}
+          <SliderFormik<Values> name="ram" label="Slider's label" min={0} max={100} />
+        </Form>
+      )}
+    </Formik>
+  )
+}
+
+export const SliderMultiRangeInFormik = () => {
+  type Values = {
+    ramRange: [number, number]
+  }
+  return (
+    <Formik<Values>
+      initialValues={{
+        ramRange: [3, 10],
+      }}
+      initialErrors={{
+        ramRange: 'Server Error: Invalid ram rannge for this cloud provider',
+      }}
+      validate={(values) => {
+        const errors: Partial<{ [key in keyof Values]: string }> = {
+          ramRange: values.ramRange[0] < 4 ? 'RAM is invalid' : undefined,
+        }
+
+        return errors
+      }}
+      onSubmit={(values) => logger.log('submit', values)}
+    >
+      {({ values }) => (
+        <Form>
+          Values: {JSON.stringify(values)}
+          <SliderFormik<Values> name="ramRange" label="Slider's label" min={0} max={100} />
+        </Form>
+      )}
+    </Formik>
+  )
+}
+
+/**
+ * The following stories are here mostly mostly for visual regression testing and do not demonstrate the way
+ * how this component should be used.
+ * @constructor
+ */
+export const DoNotUse__OpacityThumb__Beginning = () => {
```

Comment:
What exactly do they test? I do not quite get it

Reply:
When implementing a slider there are these issues that I run into https://css-tricks.com/multi-thumb-sliders-particular-two-thumb-case/#issues

These screenshots test that whoever touches this component has it done correctly, one needs to try to combine thumb height and other variables and it still work correctly. I'll add some more comments there to explain.

Reply:
Could you leave a comment with the link?